stdlib/printexc.mli

```diff
@@ -193,7 +193,7 @@ val backtrace_slots : raw_backtrace -> backtrace_slot array option
     recent function call, raise, or primitive [get_backtrace] call in
     the trace.
 
-    Some possible reasons for returning [None] are as follow:
+    Some possible reasons for returning [None] are as follows:
```

Comment:
That's not a sentence (though I'm not myself entirely convinced with the style rule when applied to technical docs). At some point, my test builds will finish too......